renovate.json

```
@@ -10,6 +10,10 @@
   "labels": ["dependencies"],
   "stopUpdatingLabel": "on hold",
   "packageRules": [
+    {
+      "matchDepTypes": ["devDependencies"],
```

Could we also exclude major dev dependency bumps from automerging? 🤔 
https://docs.renovatebot.com/configuration-options/#automerge
```suggestion
      "matchDepTypes": ["devDependencies"],
      "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
```